sdlf-ddk-lightweight/app.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from typing import Any, Dict
+import aws_cdk as cdk
+from aws_ddk_core.cicd import CICDPipelineStack
+#from utils.application_config import GetApplicationParameters
```

Comment:
Is this leftover from something? Or do we plan to use that method at some point?

Reply:
Will remove, no longer needed